docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Core_Management.adoc

```diff
@@ -132,3 +132,37 @@ Now we can list the last configuration changes :
     }]
 }
 ----
+
+[[unstable-api-annotation]]
+== [Preview Feature] Reporting of API elements annotated with annotations indicating unstable API
+
+Note: This feature only exists at the `preview` stability level or lower.
+
+Some libraries use annotations on classes/interfaces/members to indicate that the API is considered to be unstable, and likely to change or go away in a future release. For example Hibernate uses the `org.hibernate.Incubating` annotation to mark such API elements.
+
+WildFly can scan for usage of such API when your applications are deployed into WildFly, but it needs to be enabled via addition of the `/subsystem=core-management/service=unstable-api-annotations` resource. Once enabled, you can choose whether usage of these API elements should result in a warning in the logs, or a failed deployment. The next two child sections show examples of how to configure the scanner.
```

Comment:
s/for usage/for the use/g
s/usage/the use/g

Perhaps this is my personal preference, as using 'usage' as you have here is AFAICT technically correct per at least some definitions I find. But it always feels off to me, like it's an atypical way to use that word.